mail-client/alot/alot-0.9.1.ebuild

```diff
@@ -3,7 +3,8 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
Tests fail with py3.8 because notmuch does not have py3.8 yet. This is why the RESTRICT below?

Reply:
Nevermind: https://github.com/gentoo/gentoo/pull/16668 will add py3.8 to notmuch

Reply:
Cool